### PR TITLE
Feature #13 Email notification functionality

### LINKED ISSUE #13: Email notification functionality
Implement a java method that can send out an email with custom content

### PR DESCRIPTION
**Pull request by JoakimSkoog97**
Closes #13 

This commit add the email functionality. The pom file have been
updated to include the new dependencies and a file named
email.java have been created. The functionality of email.java is that it can send
emails to any email adress from a newly created email adress.

This functionality is yet to be connected to the CI server.

Co-authored-by: Robin Wänlund <robin.wanlund@hotmail.com>

### CODE REVIEW

File: CI/src/main/java/group1/email.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        properties.put("mail.smtp.ssl.enable", "true");
+        properties.put("mail.smtp.auth", "true");
+
+        // Get the Session object.// and pass username and password
+        Session session = Session.getInstance(properties, new javax.mail.Authenticator() {
+
+            protected PasswordAuthentication getPasswordAuthentication() {
+                //konto: group1SoffaKth@gmail.com
+                //losen: group1arbast!QWE
+                return new PasswordAuthentication("group1SoffaKth@gmail.com", "group1arbast!QWE");
```
Comment (olofeldre) [inline CI/src/main/java/group1/email.java:48]: no passwords please

Comment (olofeldre) [reply CI/src/main/java/group1/email.java:48]: change your password, make the code read the password from a file, and add this file to the gitignore

**Review by olofeldre — CHANGES_REQUESTED** [review]

File: CI/src/main/java/group1/email.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+import javax.mail.MessagingException;
+import javax.mail.PasswordAuthentication;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+
+public class email {
+
+    public static void main(String[] args) {
```
Comment (antonlovstrom) [inline CI/src/main/java/group1/email.java:23]: Please make this into a static function that takes a message, subject line and recipient as arguments.

**Review by antonlovstrom — CHANGES_REQUESTED** [review]

File: CI/src/main/java/group1/email.java (hunk 2 above)
Comment (antonlovstrom) [reply CI/src/main/java/group1/email.java:23]: And also please add tests to test the email functionality if possible.

**Comment by olofeldre** [comment]
All comments have been addressed in the latest commit. Please review.

**Review by CarolineLarsen — CHANGES_REQUESTED** [review]
Overall, very good! I just have a small note about the test class: all tests are not commented. Only the first test is commented properly. The second test has no comments, and the comment of the third test does not really say what the purpose of the test is.

**Comment by CarolineLarsen** [comment]
LGTM

**Pull request merged**